src/test/java/com/kodilla/ecommercee/domain/UserOrderTestSuite.java

```diff
@@ -0,0 +1,120 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.UserOrderRepository;
+import com.kodilla.ecommercee.repository.UserRepository;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class UserOrderTestSuite {
+
+    @Autowired
+    private UserOrderRepository userOrderRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    private static final String NUMBER = "101";
+
+    User user = new User(null, "Jan", "Kowalski");
```

Comment:
Można dodać private. A zamiast null w konstruktorze, to dodać konstruktor bez id w klasie User